src/lib/styles/global/colors.scss

```diff
@@ -49,33 +51,33 @@
   --cp-dark-opaque: #00000020;
   --cp-dark-gradient-1: #0f0f4d;
   --cp-dark-gradient-2: #4d1259;
-  // @deprecated (replaced with --cp-dark-50--15% or --cp-dark-1000-20?)
+  // @deprecated (same as --cp-dark-1000-20)
   --cp-dark-hover-overlay: rgba(0, 0, 0, 0.2);
   --cp-dark-1000-20: rgba(var(--cp-dark-1000-rgb), 0.2);
 
   // Accessory
-  --fade: #0E0B2660;
-  --fade-dark: #0D082680;
-  --orchid-tint: #BB5FEA25;
-  --orchid: #BB5FEA;
-  --indigo: #906BFF;
-  --indigo-tint: #906BFF25;
+  --fade: #0e0b2660;
```

Comment:
A lot of the colors you changed are not actually changed. Just case.
I think it's better not to change them in the same PR where you also change actual colors.
It makes it hard to see what actually changed and what didn't change.